dev/com.ibm.ws.webcontainer.servlet.3.1_fat/fat/src/com/ibm/ws/webcontainer/servlet31/fat/tests/WCServerTest.java

```diff
@@ -502,6 +502,15 @@ public void testServletContextCreateListenerBadListener() throws Exception {
 
     }
 
+    /**
+     * This test case verifies a plus in a URL is decoded to a space when default
+     * decodeUrlPlusSign = "true" is in effect.
+     */
+    @Test
+    public void testDecodeUrlPlusSignDefault() throws Exception {
+        this.verifyResponse("/TestServlet31/noplus+sign.html", "This file has a space in the name");
+    }
+
```

Comment:
In general I don't like the idea of  spreading related tests over multiple files it makes it very difficult to find and manage them in the future. 

You can programmatically change server configuration as well as feel free to create your own test class and a number of servers if that is more your style.

Reply:
Also I think these are more of a fat.test.mode=FULL type test rather than something we should be running in LITE mode. Any objection?

Reply:
I moved both tests to WCServerTest, set them to mode FULL.  Still working on finishing up test that runs decodeUrlPlusSign="false" case.